satellite/console/consolesim/mailsender.go

```diff
@@ -0,0 +1,84 @@
+package consolesim
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+
+	"storj.io/storj/internal/post"
+	"storj.io/storj/satellite/console"
+)
+
+// MailsSender is smtp sender mock that ignores sending process
+type MailsSender struct {
+	DB console.DB
+}
+
+// FromAddress return empty mail address
+func (sender *MailsSender) FromAddress() post.Address {
+	return post.Address{}
+}
+
+// SendEmail tries to activate account from email
+func (sender *MailsSender) SendEmail(msg *post.Message) error {
+	if msg.Subject != console.ActivationSubject {
```

Comment:
That will make it dependent of web api server which service doesn't actually require to work, or it's okay?

Reply:
I mean for testing code I think that's fine. Except why would it be dependent on the web api server, it would only need to depend on `net/http`, as far as I understand.

Reply:
Oh, I mean that it requires web server to be started and to use `console.Service`, and that this dependence is indirect.(because service doesn't know who will use it). May be I am overthinking about it:)

Reply:
So, I'm fine with depending on the web server being up and running. This means we end up testing more of our code. Of course, if there's some issue or having this approach complicates things significantly then sure... we can wire up the authorization service or database directly.

But, I understand the concern... if it doesn't work out, then it doesn't and we can use some other approach. For testing code the primary thing is about making it easy to use and ensure that changing a system doesn't require changing irrelevant tests.